spec/acceptance/itk_spec.rb

```diff
@@ -7,10 +7,8 @@
   service_name = 'apache2'
   variant = :prefork
 when 'redhat'
-  unless %r{^5}.match?(os[:release])
     variant = (os[:release].to_i >= 7) ? :prefork : :itk_only
     service_name = 'httpd'
```

Comment:
```suggestion
  variant = :prefork
  service_name = 'httpd'
```

However, with that there is nothing left that's the `itk_only` variant and that can also be cleaned up now.

Reply:
Oh I missed it. Thanks for pointing out @ekohl